### ISSUE TITLE
Mangler

### ISSUE THREAD

**Issue by lazorfuzz** [leonli96/python-hacklib#3]
@tylerp96 

Hi glad to hear back from you!

The head branch was super outdated so I just deleted it. Anyway, I checked out your mangler code and it's looking really good, except for some small bugs in your Reverse() and UniqueLeet() functions which don't take 'self' as arguments. Other than that, I'm excited to add it into hacklib!

If you don't mind, fork the current master branch and add your Mangler class and redo the pull request. We can definitely make it compatible with the topPasswords() function for more variety in password generation.

**Comment by lazorfuzz** [comment]
Also, having password generation with the mangler in the UI may be a good idea if someone just wants to generate passwords.

**Comment by r0t0tiller** [comment]
Opps! Must have missed that! Ill fix that and fork the master, and add it! Very excited for this!

**Comment by blindfuzzy** [comment]
Finally he gets something done :P JK...slow progress on our end...might be incorporating hacklib in our new recon tool [LHF](https://github.com/blindfuzzy/LHF) will make some ideas we have a tad bit easier. 

@leonli96 we are in a super busy part of the year so we are slow af atm. 
@tylerp96 good work man!

EDIT: ps i did a username change btw. -.-

**Comment by lazorfuzz** [comment]
@blindfuzzy I took a look at your LHF and it looks great! As of now it seems like hacklib might just be redundant since you already have nmap going. But hopefully one day hacklib will carve out its own niche in the LHF module set! 

On a side note, I had a lot of fun with hacklib the other day at a coffee shop. Found an HP Laserjet printer in their network with port 9100 open and used hacklib to print out some mildly offensive text ;)
